javagen/src/main/java/com/azure/autorest/template/WrapperClientMethodTemplate.java

```diff
@@ -68,7 +71,14 @@ public void write(ClientMethod clientMethod, JavaType typeBlock) {
             }
 
             writeMethodInvocation(clientMethod, function, shouldReturn);
-        });
+        };
+        if (clientMethod.getMethodVisibilityInWrapperClient() == JavaVisibility.Public) {
+            typeBlock.publicMethod(declaration, method);
+        } else if (typeBlock instanceof JavaClass && clientMethod.getMethodVisibilityInWrapperClient() == JavaVisibility.PackagePrivate) {
+            JavaClass classBlock = (JavaClass) typeBlock;
+            classBlock.packagePrivateMethod(declaration, method);
+        }
```

Comment:
this "if" should be able to be unified.

there should be a `method` that takes `JavaVisibility`.

Reply:
You mean something like this?
```java
 if (typeBlock instanceof JavaClass) {
            JavaClass classBlock = (JavaClass) typeBlock;
            classBlock.method(clientMethod.getMethodVisibilityInWrapperClient(), null, declaration, method);
 }
```

Reply:
Yes.

The point of having a JavaVisibility instead of boolean, is to use it.